.prettierignore

```diff
@@ -0,0 +1,7 @@
+# Ignore specific apps
+apps/_compability_app
```

Comment:
@Dimitreee %)

Reply:
Dimitreee asked to remove this, I've just forgotten. 
Will fix